packages/aws-cdk/lib/api/aws-auth/awscli-compatible.ts

```diff
@@ -120,19 +126,6 @@ export class AwsCliCompatible {
     };
   }
 
-  public static proxyAgent(options: SdkHttpOptions) {
```

Comment:
pulled out into a separate class. this had otherwise nothing to do with this file